com.unity.render-pipelines.core/Runtime/Lighting/ProbeVolume/ProbeReferenceVolume.cs

```diff
@@ -295,37 +295,41 @@ internal class Cell
             public SphericalHarmonicsL2[] sh;
             public float[] validity;
             public int minSubdiv;
-
-            [System.NonSerialized]
-            public int flatIdxInCellIndices = -1;
-
-            [System.NonSerialized]
-            public bool loaded = false;
+            public int indexChunkCount;
+            public int shChunkCount;
         }
 
-        class CellChunkInfo
+        [DebuggerDisplay("Index = {cell.index} Loaded = {loaded}")]
+        internal class CellInfo : IComparable<CellInfo>
         {
-            public List<Chunk> chunks;
-        }
-
-        private class CellSortInfo : IComparable
-        {
-            internal string sourceAsset;
-            internal Cell cell;
-            internal float distanceToCamera = 0;
-            internal Vector3 position;
+            public Cell cell;
+            public List<Chunk> chunkList = new List<Chunk>();
+            public int flatIdxInCellIndices = -1;
+            public bool loaded = false;
+            public ProbeBrickIndex.CellIndexUpdateInfo updateInfo;
+            public int sourceAssetInstanceID;
```

Comment:
Any reason for this change? It was very useful to use a string as a ID to debug

Reply:
Perf reason. I need to check that everytime an asset is streamed in/out to remove existing cells from internal arrays. Comparing strings did not feel good at all.
I might be overthinking this though, for now we don't even have asset streaming so...